### DISCUSSION TITLE
Tidal APIs for creating user playlist on request

### DISCUSSION THREAD

**Discussion by m-shariq** [tidal-music/discussions#36] [Q&A] [answered]
Hey there,

I have explored the internet regarding Tidal and found some APIs regarding playlist creation hosted on **api.tidal.com** but I can't use them and there's no documentation available on the internet. I can't use **openapi.tidal.com** because it doesn't support playlist, track or album creation and update.

Can you please share the relevant docs with me and how to get the access token of the user using **login.tiddle.com/authorize**? 

Thanks
@stefanmoro

**Comment by stefanmoro** [comment, marked as answer]
Hey @m-shariq,

TIDAL Developer Platform does not support accessing TIDAL user's playlists just yet, this is something we are working on supporting though, and we hope to share updates about this this spring.

The APIs you are referring to that support playlist creations vi api.tidal.com represents represents an unauthorized usage of TIDALs API. Please see https://github.com/orgs/tidal-music/discussions/7 (and [this comment](https://github.com/orgs/tidal-music/discussions/7#discussioncomment-7124801) in particular) for further information.

We will post any updates here as soon as we make more parts of our platform available for usage by 3rd party application.

**Reply by m-shariq** [reply]
Thank you and looking forward
